terraform/helm/aptos-node/files/configs/validator-base.yaml

```diff
@@ -21,6 +21,11 @@ consensus:
           from_file: /opt/aptos/genesis/waypoint.txt
         identity_blob_path: /opt/aptos/genesis/validator-identity.yaml
 
+state_sync:
+  data_streaming_service:
```

Comment:
I dont think we want to commit this as a default @zekun000 @JoshLind

Reply:
The commit says [DO NOT COMMIT] :)